shell-completion/bash/firewall-offline-cmd

```diff
@@ -0,0 +1,228 @@
+# bash completion for firewall-offline-cmd                         -*- shell-script -*-
+
+# Authors:
+# Danilo Figueiredo Rocha <drocha.figueiredo@gmail.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+# TODO: find a way how to get the following options from firewall-offline-cmd
+
+## Revisado todas as opções --DFR-- 30-07-2018
```

Comment:
Please use English comments. These look stale anyway. Maybe just remove them.

Reply:
Sorry, I just forgot to remove, this comment is useless.

Can you remove this? or what I have to do to fix my PR?

Reply:
You can remove these comments. You'll have to push other changes anyway.